tofu/main/aws/variables.tf

```diff
@@ -135,7 +135,7 @@ variable "tester_cluster" {
     server_count   = 1
     agent_count    = 0
     distro_version = "v1.26.9+k3s1"
-    public_ip = true
+    public_ip = false
```

Comment:
Is this cluster still accessible via `tester.local.gd` when this is false?

Reply:
Sure, that's the default